### DISCUSSION TITLE
Password for QBitTorrent

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#3616] [Help Needed]
Hi

I install qbittorent LXC and it says password wrong. I tried admin/changeme and admin/adminadmin and neither worked. Could someone help me please? I don't know how to reset it, and ecosia wasn't much help in that regard.

TIA

**Comment by ghost** [comment]
sorry, I figured out how to reset the password. You can close now :)

**Reply by Chewyyy** [reply]
Could you provide how you did this?

**Reply by oOStroudyOo** [reply]
I use qBittorent via Docker so sorry if I'm slightly wrong but based on the Tteck script it should be something like this:

Open the qBittorent config file:
`nano /.config/qBittorrent/qBittorrent.conf`

Then scroll down to WebUI\Password and replace it with:
`WebUI\Password_PBKDF2="@ByteArray(ARQ77eY1NUZaQsuDHbIMCA==:0WMRkYTUWVT9wVvdDtHAjU9b3b7uB8NR1Gur2hmQCvCDpm39Q+PsJRJPaCU51dEiz+dTzh8qbPsL8WkFljQYFQ==)"`
This will apparently set the password to adminadmin

Scroll down to WebUI\Username to change your username if you wish.

Save the config file `Ctrl + O`
Exit `Ctrl + X`

Reboot qBittorrent container.

Your new username and the adminadmin password should be active, then change your password to something you prefer on qBittorent via Settings > Web UI > Authentication > Username/Passwd > Save

**Reply by Chewyyy** [reply]
I’m running it though a proxmox LXC, which you have to edit the conf file while the container is running. 

But I think your method works, BUT, the UI needs to be logged into using a private browser (I guess qbitt caches on the browser, so when resetting the password, it appears like it doesn’t actually reset). 

But once you do this, restart the container, and then login through the UI on private browsing, it’ll let you use the stock password. 

Still doesn’t explain why my password keeps resetting, even after I set it.

**Reply by oOStroudyOo** [reply]
I think you're right about the caching, I've witnessed this when changing themes.

Are you saying that even if you set the password via the browser once you're logged in, it's resetting it back to the default password?
Or are you setting it via the config file?

**Reply by Chewyyy** [reply]
Resetting in the UI once I’m logged in, it’ll randomly reset on me, no clue why.
